docs/definition.rst

```diff
@@ -1,27 +1,94 @@
-Execution Environment Definition
+Execution environment definition
 ================================
 
-The execution environment (EE) definition file supports multiple versions.
+You define the content of your execution environment in a YAML file. By default, this file is called ``execution_environment.yml``. This file tells Ansible Builder how to create the containerfile (Containerfile for Podman, Dockerfile for Docker) and build context for your container image. The execution environment definition file accepts four top-level sections: version, images, dependencies, and (optionally) additional_build_steps.
```

Comment:
I prefer something like `build instruction file` or even just `instruction file` over `containerfile`.

Reply:
Changed, thanks for the feedback.